aeppl/transforms.py

```diff
@@ -272,7 +272,7 @@ def create_default_transformed_rv_op(
 )
 TransformedHalfNormalRV = create_default_transformed_rv_op(
     at.random.halfnormal,
-    LogTransform(),
+    IntervalTransform(lambda *inputs: (inputs[3], None)),
```

Comment:
I find this way of indexing into `input` very hard to read.

Reply:
Would `IntervalTransform(lambda *_, loc, sigma: (loc, None))` be better?

Reply:
I think better would be if we unpacked them `rv, op, ... = inputs`. Or turn inputs into a dict or NamedTuple.

Reply:
The first 3 inputs are always the same, it would be `rng, dtype, size, inp1, inp2, inpN = inputs` but that adds a lot of noise I think. I don't see where the named tuples could be introduced.

Alternatively we could pass the argument indexes ignoring the first three:

`IntervalTransform(lower_arg_idx=0, upper_arg_idx=None)`, which is perhaps easier to read than having to subtract 3 to know which actual "input" it is.

Reply:
Why not do `rng, dtype, size, *inp = inputs`?

Reply:
Seems strange to type out explicitly only those variables that we will not use

Reply:
I like `IntervalTransform(lambda *_, loc, sigma: (loc, None))`.

Reply:
Actually that's not allowed, `loc` and `sigma` would become keyword only arguments...

Reply:
OK well maybe just leave it and add a comment. Or go with your option without the lambda.

Reply:
Went with a comment

Reply:
Looks good!